Source/radiation/MGFLD.cpp

```diff
@@ -1670,13 +1670,41 @@ void Radiation::inelastic_scattering(int level)
                 const Box& bx = mfi.tilebox();
 
                 kps.resize(bx,1); // we assume scattering is independent of nu
+                Elixir elix_kps = kps.elixir();
                 MGFLD_compute_scattering(kps, S_new[mfi]);
 
-                ca_inelastic_sct(ARLIM_3D(bx.loVect()), ARLIM_3D(bx.hiVect()),
-                                 BL_TO_FORTRAN_ANYD(S_new[mfi]),
-                                 BL_TO_FORTRAN_ANYD(Er_new[mfi]),
-                                 BL_TO_FORTRAN_ANYD(kps),
-                                 dt);           
+                const auto state = S_new[mfi].array();
+                const auto kps_arr = kps.array();
+                const auto Er_arr = Er_new[mfi].array();
+
+                amrex::ParallelFor(bx,
+                [=] AMREX_GPU_HOST_DEVICE (int i, int j, int k)
+                {
+
+                    Real temp = state(i,j,k,UTEMP);
+                    Real ks = kps_arr(i,j,k);
+
+                    Real dEr = 0.0;
+
+                    Real Er[NGROUPS];
+                    for (int g = 0; g < NGROUPS; ++g) {
+                        Er[g] = Er_arr(i,j,k,g);
+                    }
+
+                    ca_inelastic_sct(i, j, k,
```

Comment:
This call won't work in the GPU build; how much extra work is it to finish the GPU port of the inelastic scattering function?

Reply:
it is ugly and sort of orthogonal to my goal here.  Maybe I'll just remove the GPU tag.

Reply:
removed from the GPU for now

Reply:
switched to `LoopOnCpu`